code/modules/projectiles/guns/lawgiver.dm

```diff
@@ -173,43 +173,43 @@
 		if(dna_profile == H.dna.unique_enzymes)
 			recoil = 0
 			if((findtext(speech.message, "stun")) || (findtext(speech.message, "taser")))
-				firing_mode = STUN
+				firing_mode = LAWGIVER_STUN
 				fire_sound = 'sound/weapons/Taser.ogg'
 				projectile_type = "/obj/item/projectile/energy/electrode"
 				fire_delay = 0
 				sleep(3)
-				say("STUN.")
+				say("LAWGIVER_STUN.")
```

Comment:
![image](https://user-images.githubusercontent.com/31417754/45846136-87cba300-bd16-11e8-9f86-7d0767ede3f4.png)

I'm not sure if this will work.
Also, the Lawgiver is supposed to say those in all caps

Reply:
ebin, this one's a mistake